tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Redis/ServiceStack/RedisNativeClientSendReceiveIntegration_6_2_0.cs

```diff
@@ -0,0 +1,102 @@
+// <copyright file="RedisNativeClientSendReceiveIntegration_6_2_0.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Redis.ServiceStack
+{
+    /// <summary>
+    /// ServiceStack.Redis.RedisNativeClient.SendReceive[T] calltarget instrumentation (v6.2.0+)
+    /// <para>
+    /// 6.2.0 of ServiceStack.Redis changed the signature of SendReceive (added string parameter).
+    /// See: https://github.com/ServiceStack/ServiceStack/commit/950155db0d1b358920eab307c61e334e24f1d174
+    /// Also, 6.2.0 broke .NET Core 2.1 and .NET Core 3.0 compatibility as ServiceStack
+    /// accidentally added a dependency that claimed it was .NET Standard 2.0 compliant
+    /// but in reality wasn't.
```

Comment:
This comment is helpful, but maybe it could be placed somewhere else where we build/test? My first thought was adding it to `tracer/build/PackageVersionsGeneratorDefinitions.json` but I don't know if we can add comments there without breaking our JSON parsing 😞 I think the instrumentation here is only concerned with the method signature and the assembly dependency issue that was also introduced in 6.2.0 is a separate concern

Reply:
Yeah I didn't really know where to put this, but it doesn't make a ton of sense here

I didn't think of attempting to add a comment to the `ackageVersionsGeneratorDefinitions.json`, so I'll try that and maybe also adding it as a comment in the sample csproj as well?

Reply:
So I tossed a variation of the comment in the sample project.

I also learned that comments just aren't allowed in JSON whatsoever.